app/forms/journeys/additional_payments_for_teaching/claim_submission_form.rb

```diff
@@ -0,0 +1,53 @@
+module Journeys
+  module AdditionalPaymentsForTeaching
+    class ClaimSubmissionForm < ::ClaimSubmissionBaseForm
+      private
+
+      def selected_claim_policy
+        case answers.selected_policy
+        when "EarlyCareerPayments"
+          Policies::EarlyCareerPayments
+        when "LevellingUpPremiumPayments"
+          Policies::LevellingUpPremiumPayments
+        when nil
+          nil
+        else
+          fail "Invalid policy selected: #{answers.selected_policy}"
+        end
+      end
+
+      def main_eligibility
+        @main_eligibility ||= eligibilities.detect { |e| e.policy == main_policy }
+      end
+
+      def main_policy
+        if selected_claim_policy.present?
+          selected_claim_policy
+        else
+          Policies::EarlyCareerPayments
+        end
+      end
```

Comment:
It looks like we can remove this, I've opened a ticket to do so https://dfedigital.atlassian.net.mcas.ms/browse/CAPT-1696 as it requires updating a bunch of tests that have already been changed in branches based off this, so it will be much easier to do once some other work has been merged